languages/java/build.gradle

```diff
@@ -19,14 +19,14 @@ repositories {
         api 'net.java.dev.jna:jna-platform:5.12.1'
     }
 
-    description = 'BitwardenSDK'
+    description = 'Bitwarden Secrets Manager Java SDK'
     java.sourceCompatibility = JavaVersion.VERSION_1_8
 
     publishing {
         publications {
             maven(MavenPublication) {
                 groupId = 'com.bitwarden'
-                artifactId = 'sdk'
+                artifactId = 'sdk-secrets'
```

Comment:
i though this sdk repo is not specific to secrets manager, but it's for everyone ?